.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,22 @@
+name: Enhancement request
+description: Do you have a suggestion or feature request?
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to propose a new feature or make a suggestion.
+  - type: textarea
+    id: suggestion
+    attributes:
+      label: Suggestion
+      description: Please feel welcome to include screenshots or sketches.
+      placeholder: Tell us what you would like to do!
+      value: |
+        #### What would you like to do?
+
+        #### Why would you like to do it?
+
+        #### How would you like to achieve it?
```

Comment:
Can this be a placeholder?

Reply:
Should I split this into multiple text areas too?

Reply:
No

Reply:
Ok, do you want me to replace the existing placeholders with these three lines or just the last?

Reply:
Actually, just remove the value here and make the placeholder "I would like to ..."

Reply:
You can use the suggestion button to show me what you mean.

When you respond on GitHub, this one: ![Suggestion button](https://user-images.githubusercontent.com/6966049/169598058-4b6db03a-70a2-4375-8f92-6fd7255dca96.png)